lib/sidekiq_unique_jobs/sidekiq_worker_methods.rb

```diff
@@ -37,9 +37,9 @@ def worker_class
     def after_unlock_hook
       lambda do
         if @worker_class.respond_to?(:after_unlock)
-          @worker_class.after_unlock # instance method in sidekiq v6
+          @worker_class.after_unlock(item) # instance method in sidekiq v6
```

Comment:
I love the suggestion, in this case we would likely have to check the arity of the method and only pass the item if the callback method responds to 1 method argument and warn if it does not. 

This would otherwise crash for a lot of users if the method doesn't support an argument?

Reply:
You are right.  I wasn't aware of the `arity` check possibility.  I added it.
The PR is blocked on a code climate issue.  Breaking up the method seems to add more complexity...